packages/sdk/src/types/Qualifications.ts

```diff
@@ -0,0 +1,109 @@
+import { CustomerRequest } from './Customers'
+import { Junction } from './Exports'
+import { OrderRequest, OrderResponse } from './Orders'
+import { ApplicableToResultList, InapplicableToResultList } from './ApplicableTo'
+import { Category } from './Categories'
+import { Discount } from './DiscountVoucher'
+import { LoyaltiesTransferPoints } from './Loyalties'
+import { ValidationRulesAssignmentsList } from './ValidationRules'
+
+export type QualificationsCheckEligibilityRequestBody = {
+	customer?: CustomerRequest
+	order?: OrderRequest
+	mode?: 'BASIC' | 'ADVANCED'
+	tracking_id?: string
+	scenario?:
+		| 'ALL'
+		| 'CUSTOMER_WALLET'
+		| 'AUDIENCE_ONLY'
+		| 'PRODUCTS'
+		| 'PRODUCTS_DISCOUNT'
+		| 'PROMOTION_STACKS'
+		| 'PRODUCTS_BY_CUSTOMER'
+		| 'PRODUCTS_DISCOUNT_BY_CUSTOMER' // default ALL
+	options?: {
+		limit?: number //max 100
+		starting_after?: string
+		filters?: {
+			junction?: Junction
+		} & Partial<Record<QualificationsFiltersFields, QualificationsFieldConditions>>
+		expand?: ('redeemable' | 'category' | 'validation_rules')[]
+		sorting_rule?: 'BEST_DEAL' | 'LEAST_DEAL' | 'DEFAULT'
+	}
+	metadata?: Record<string, any>
+}
+
+export type QualificationsCheckEligibilityResponseBody = {
+	redeemables: QualificationsRedeemablesResponse
+	tracking_id?: string
+	order?: OrderResponse
+	stacking_rules: QualificationsStackingRulesResponse
+}
+
+//domain type
+export type QualificationsFiltersFields =
+	| 'category_id'
+	| 'campaign_id'
+	| 'resource_id'
+	| 'resource_type'
+	| 'voucher_type'
+	| 'code'
+
+export type QualificationsFiltersCondition = '$in' | '$not_in' | '$is' | '$is_not' | '$has_value' | '$is_unknown'
+
+export type QualificationsFieldConditions = { conditions?: Partial<Record<QualificationsFiltersCondition, any>> }
+
+export type QualificationsRedeemablesResponse = {
+	object: 'list'
+	data_ref: 'data'
+	data: QualificationsRedeemableSingleResponse[]
```

Comment:
Shouldn't we just call it: QualificationsRedeemables (instead of QualificationsRedeemableSingleResponse)?
You can see that `QualificationsRedeemablesResponse` actually points to a response (has object, data_ref), while `QualificationsRedeemableSingleResponse` can be treated as a so-called "domain type", it would be better imho